app/views/layouts/application.html.erb

```diff
@@ -12,13 +12,14 @@
 
 <script>
   $(function() {
+    var ideas = $('.ideas');
     $('form#createIdea').submit(Ideas.create);
```

Comment:
I'm not sure if it's necessary to save this selector as a variable since it's not being repeated